components/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/RequestExecutor.java

```diff
@@ -108,6 +150,97 @@ private void validateExecutionParams() throws InternalWorkflowException {
             throw new InternalWorkflowException("Init params for the BPELExecutor is null.");
         }
     }
+    private String getExternalWorkflowId(String workflowId) throws WorkflowException {
```

Comment:
Please check new lines between methods, new line in the 1st line of the method body and remove unnecessary new lines